src/Domain/Creators/TimeDependentOptions/ShapeMap.hpp

```diff
@@ -262,5 +262,5 @@ template <bool IncludeTransitionEndsAtCube, domain::ObjectLabel Object>
 std::pair<std::array<DataVector, 3>, std::array<DataVector, 4>>
 initial_shape_and_size_funcs(
```

Comment:
Can't reach with GitHub comments, but we should note somewhere in some help text that with this change, the deformed radius will be exactly what is in the YlmsFromFile, and not what is set for the inner/outer radius in the input file.